code/modules/mob/living/carbon/human/species/abductor.dm

```diff
@@ -24,4 +24,7 @@
 	H.gender = NEUTER
 	if(H.mind)
 		H.mind.abductor = new /datum/abductor
-	return ..()
\ No newline at end of file
+	H.remove_language("Tradeband")
```

Comment:
Couldn't you Cut() their `languages` list and then just add whatever the abductor language is?